src/Symfony/Component/HttpKernel/CacheWarmer/CacheWarmerAggregate.php

```diff
@@ -93,12 +95,17 @@ public function warmUp(string $cacheDir): array
                     continue;
                 }
 
+                $start = microtime(true);
                 foreach ((array) $warmer->warmUp($cacheDir) as $item) {
                     if (is_dir($item) || (str_starts_with($item, \dirname($cacheDir)) && !is_file($item))) {
                         throw new \LogicException(sprintf('"%s::warmUp()" should return a list of files or classes but "%s" is none of them.', $warmer::class, $item));
                     }
                     $preload[] = $item;
                 }
+
+                if ($io?->isDebug()) {
+                    $io->info(sprintf('"%s" finished warmup in %.2f ms.', $warmer::class, (microtime(true) - $start) * 1000.0));
```

Comment:
`$io->info(sprintf('"%s" completed in %.2fms.', $warmer::class, 1000 * (microtime(true) - $start)));`

Reply:
`%0.2f` actually?

Reply:
Suits me fine, that's updated!